Submitted By:            Igor Živković
Date:                    2014-06-26
Initial Package Version: 10.2.2
Upstream Status:         Rejected
Origin:                  http://patchwork.ozlabs.org/patch/346792/ and
                         http://patchwork.freedesktop.org/patch/22673/
Description:             Checks for libudev only for dri3.

diff -Naur Mesa-10.2.2.orig/configure.ac Mesa-10.2.2/configure.ac
--- Mesa-10.2.2.orig/configure.ac	2014-06-25 06:33:39.000000000 +0200
+++ Mesa-10.2.2/configure.ac	2014-06-26 21:45:38.630169401 +0200
@@ -817,8 +817,6 @@
 fi
 
 case "$host_os" in
-linux*)
-    need_libudev=yes ;;
 *)
     need_libudev=no ;;
 esac
@@ -1011,11 +1009,11 @@
         DEFINES="$DEFINES -DHAVE_ALIAS"
         if test "x$enable_dri3" = xyes; then
             DEFINES="$DEFINES -DHAVE_DRI3"
-        fi
 
         if test "x$have_libudev" != xyes; then
             AC_MSG_ERROR([libudev-dev required for building DRI])
         fi
+        fi
 
         case "$host_cpu" in
         powerpc* | sparc*)
